README.md

```diff
@@ -56,3 +56,42 @@ Please see the sample in the `Common` class: https://github.com/keboola/db-extra
 The namespace of your extractor class shoud be `Keboola\DbExtractor\Extractor` and the name of the class should corespond to DB vendor name i.e. PgSQL, Oracle, Impala, Firebrid, DB2 and so on.
 
 Please check the existing implementations above for help getting started.
+
+## Check username
+
+Check username functionality compares 
+    - value of the `KBC_REALUSER` environment variable 
+    - and username of the database user from the configuration.
+
+If enabled, values must be same to run the component.
+
+If a service account is used, check is skipped. For more information see `UsernameChecker` class.
+
+**Example image / stack parameters**
+```json
+{
+  "checkUsername": {
+    "enabled": true
+  }
+}
+```
+
+Service account can be defined by regular expression:
+```json
+{
+  "checkUsername": {
+    "enabled": true,
+    "serviceAccountRegexp": "~^service_~i"
+  }
+}
+```
+
+Or user account can be defined by regular expression (inverted logic):
+```json
+{
+  "checkUsername": {
+    "enabled": true,
+    "userAccountRegexp": "~^[0-9]{3}_"
```

Comment:
Rozhodlo sa, ze uzivatelsky ucet sa bude detekovat pomocou regularneho vyrazu:
https://keboola.slack.com/archives/CQ1ASK06M/p1612969524046200

Dal som to do stack parameters, kedze si nie som uplne isty, ze sa to o par dni zase nezmeni.

Reply:
Do stack parameters dam regexp `userAccountRegexp:` -> `~^[a-z][a-z0-9]{2}[0-9]{5}$~i`, skontroluj prosim.
https://regex101.com/r/J8uPWb/1

Reply:
padak psal `^[A-Z][A-Z0-9]{2}[0-9]{5}$` ty tu povoluješ i malé písmena....spíš bychh to zachoval přesně jak psali

Reply:
@ondrajodas 

padak napisal `upper(UserID) regexp like  '^[A-Z][A-Z0-9]{2}[0-9]{5}$'   ==> jde o jmenný (osobní účet), vše ostatní budeme považovat za technický.` 

Teda to naprv previdelo na `upper`, ... to je to iste ako dat tam parameter `i` ... teda velke aj male.

Reply:
@webrouse 🎉 no jasně, toho upperu jsem si nevšiml :D